index.html

```diff
@@ -0,0 +1,732 @@
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="contact-layout">
+        <div class="contact-layout__contents">
+          <div class="contact-layout__sub-title">Contact us</div>
+          <div class="contact-layout__title">Make an appointment</div>
+          <div class="contact-layout__input-container">
+            <div class="contact-layout__input-container1">
+              <input
+                class="contact-layout__input"
+                type="text"
+                placeholder="Full Name"
+              />
+              <input
+                class="contact-layout__input"
+                type="text"
+                placeholder="example@gmail.com"
+              />
+            </div>
+            <div class="contact-layout__input-container1">
+              <select class="contact-layout__input">
+                <option value="">Please Select</option>
+                <option value=""></option>
+              </select>
+              <select class="contact-layout__input">
+                <option value="en">4:00 Available</option>
+                <option value=""></option>
+              </select>
+            </div>
+            <div class="contact-layout__input-message">
+              <textarea
+                class="contact-layout__input-messages"
+                name=""
+                id=""
+                cols="30"
+                rows="10"
+              >
+Message</textarea
+              >
+            </div>
+            <div>
+              <button class="button button--link">Book an appointment</button>
+            </div>
+          </div>
+        </div>
+        <div class="contact-layout__agency">
+          <div class="contact-layaut__agency-content">
+            <h1 class="contact-layout__agency-title">
+              Consulting Agency For Your Business
+            </h1>
+            <p class="contact-layout__agency-subtitle">
+              the quick fox jumps over the lazy dog
+            </p>
+          </div>
+          <div>
+            <button class="contact-layout__agency-button">Contact Us</button>
+          </div>
+        </div>
+      </div>
+    </main>
+    <footer class="footer">
+      <div class="footer__content-container">
+        <div class="footer__contents">
+          <div class="footer__logo">KMarketer</div>
+          <div class="footer__socials">
+            <img src="./images/face.svg" alt="facebook-icon" />
+            <img src="./images/insta.svg" alt="insta-icon" />
+            <img src="./images/twitte.svg" alt="twitte-icon" />
+            <img src="./images/youtube.svg" alt="utube-icon" />
+          </div>
+        </div>
+        <div class="footer__border"></div>
+
+        <div class="footer__menu-item">
+          <div class="footer__menus">
+            <div class="footer__title">Company Info</div>
+            <div class="footer__menu">About Us</div>
```

Comment:
same here you can improve your namings with this classes `footer__menu` `footer__menus`